server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -142,11 +161,15 @@ export function create(
 		const tenantCustomData: ITenantCustomData = request.body.customData
 			? request.body.customData
 			: {};
+		const enablePrivateKeyAccess = request.body.enablePrivateKeyAccess
+			? request.body.enablePrivateKeyAccess
+			: null;
```

Comment:
Nit: in the client codebase we try not introduce any new uses of `null`, either preferring `undefined` or an actual value. I think setting `null` here just makes it so the call to `manager.createTenant()` uses its default value of `false`. Would it be better to make that parameter required, and use `false` as the fallback here instead?

Nit 2: I'd use `const enablePrivateKeyAccess = request.body.enablePrivateKeyAccess ?? false`. Besides the conciseness, I think it might make it easier to think of this assignment. If `request.body.enablePrivateKeyAccess = false` in the request, this is currently converting it to `null`, and then letting the default param value in `createTenant()` take over, but it seems better to just respect the `false` from the request in that case?